duckduckgo/macos-browser
# Remove the Never option from the Save Credentials view controller

Task/Issue URL: https://app.asana.com/0/1177771139624306/1201780988356238/f
Tech Design URL:
CC: @brindy 

**Description**:

This PR removes the Never option from the Save Credentials view controller.

**Steps to test this PR**:
1. Run a `develop` build of the app, and sign into some websites, selecting the Never option each time
1. Check out this branch, and check after launch that user defaults has been cleared (`defaults read com.duckduckgo.macos.browser.debug`)
1. Sign into some websites and check that the `Never` option isn't there

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
